src/account/useAnonSignIn.ts

```diff
@@ -0,0 +1,23 @@
+"use client";
+
+import { useMutation, useQueryClient } from "@tanstack/react-query";
+import { Models } from "appwrite";
+import { useAppwrite } from "react-appwrite";
+
+function useAnonSignIn() {
```

Comment:
Can you rename it to `useAnonymousSignIn`? I think it is better to name things with their full name and to only shorten when it makes the most sense, like with OAuth for example.